src/tests/index.test.ts

```diff
@@ -189,8 +189,8 @@ describe('Move Artifacts', () => {
     });
 
     it('should skip function if skipEsbuild is set to true', async () => {
-      // @ts-ignore
```

Comment:
This was giving a warning whenever running tests; the fix is to use `jest.mocked` to wrap the function to get the mock types to work correctly.